sqlite3_vendor/CMakeLists.txt

```diff
@@ -15,8 +15,8 @@ endif()
 ament_vendor(sqlite3_vendor
   SATISFIED ${SQLite3_FOUND}
   VCS_TYPE zip
-  VCS_URL https://www.sqlite.org/2022/sqlite-amalgamation-3370200.zip
-  VCS_VERSION sqlite-amalgamation-3370200
+  VCS_URL https://www.sqlite.com/2024/sqlite-amalgamation-3450300.zip
+  VCS_VERSION ssqlite-amalgamation-3450300
```

Comment:
Changing the URL here is suspicious. This should remain sqlite.org .  Also, there is a typo in that there should only be one `s` in the VCS_VERSION.